### PR TITLE
Rewrite the plugin in ES2015

### PR DESCRIPTION
**Pull request by reenko**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -30,21 +30,22 @@
...
 module.exports = {
+
 	/**
 	 * Registers all localization components in service locator.
 	 * @param {ServiceLocator} locator Catberry's service locator.
 	 */
-	register: function (locator) {
-		var config = locator.resolve('config');
+	register(locator) {
+		const config = locator.resolve('config');
```
Comment (rdner) [inline index.js]: `config` is not used anywhere.

File: lib/LocalizationHelper.js [hunk 2]
```diff
@@ -30,68 +30,68 @@
...
- * @param {LocalizationProvider} $localizationProvider Localization provider.
- * @constructor
- */
-function LocalizationHelper($localizationProvider) {
-	this._localizationProvider = $localizationProvider;
-}
+	/**
+	 * Creates new instance of localization helper.
+	 * @param {ServiceLocator} locator Locator to resolve dependencies.
+	 * @constructor
```
Comment (rdner) [inline lib/LocalizationHelper.js]: Remove this annotation.

File: lib/LocalizationHelper.js [hunk 3]
```diff
@@ -30,68 +30,68 @@
...
-LocalizationHelper.prototype.getHandlebarsHelper = function () {
-	return function () {
-		var key = null,
-			locale = null,
-			count = null,
-			options = arguments[arguments.length - 1];
+		/* eslint prefer-rest-params:0 */
+		/* eslint no-invalid-this:0 */
+		// TODO rewrite this when Spread operator and Rest parameters will be supported
+		return function() {
```
Comment (rdner) [inline lib/LocalizationHelper.js:52]: We don't need `.bind(this)` here, an arrow function should work instead.

Comment (reenko) [inline lib/LocalizationHelper.js:52]: We can't use arrow function because we use arguments and in arrow function is empty

Comment (rdner) [inline lib/LocalizationHelper.js:52]: Yes, you're right, did notice `arguments` usage.

**Pull request merged**